portal/src/components/Search/Search.tsx

```diff
@@ -0,0 +1,200 @@
+                                                        ></path>
+                                                    </g>
+                                                </svg>
+                                            </a>
+                                        </div>
+                                    </div>
+                                );
+                            },
+                        },
+                    },
+                ]),
+        });
+
+        const focusSearcInputListener = (event: globalThis.KeyboardEvent) => {
+            if (event.key === "/") {
```

Comment:
Følg etter GitHub og NPM og bruk denne som snarvei for å fokusere på søket

Reply:
Jeg tror vi kunne jobbe lit mer med stiling av søkfeltet.

- Defaulttilstand gjennomsiktig
- For meg er ikke den aktive tilstanden riktig ennå. det ser ut som 3 linjer ved inntastingsfeltet.
- Hoverstate mangler
- Kan vi fjerne Logo fra algolia?
- Jeg vet heller ikke om vi virkelig må vise knappefunksjonen

Reply:
Funker dårlig i Firefox. Der er dette snarveien for "finn på siden" (passende nok), så det overstyrer 😅 Er nok også en mer praktisk snarvei på tastaturer med engelsk layout, siden det er en egen tast. Men har ikke umiddelbart noe bedre forslag!